src/stream/src/task/stream_manager.rs

```diff
@@ -585,23 +546,20 @@ impl LocalStreamManagerCore {
         Ok((executor, subtasks))
     }
 
-    async fn build_actors(
-        &mut self,
-        actors: &[ActorId],
+    async fn build_actors_impl(
```

Comment:
Just curious, why we suffix with `_impl` here, why not keep the name as is.

Reply:
A method of the same name already exists in `LocalStreamManager`. This method was in `LocalStreamManagerCore` so there was no naming conflict. This PR changes it to a method of `LocalStreamManager`.